### PR TITLE
Updated version of Django and xhtml2pdf

### PR DESCRIPTION
**Pull request by DotSlashTX**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-django>=2.0,<2.2
+django>=2.0.6,<2.2
 swapper>=1.1.0,<1.2.0
 openwisp-utils>=0.2,<0.3
 djangorestframework>=3.8.2,<3.9
 passlib>=1.7.1,<1.8.0
 django-filter>=2.0.0,<2.1.0
 djangorestframework-link-header-pagination>=0.1.1,<0.2.0
-xhtml2pdf==0.2.2
+xhtml2pdf>=0.2.1,<0.2.2
```
Comment (rohithasrk) [inline requirements.txt]: Hi. We are supposed to update the version to >=0.2.3, <0.3.0.

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
-django>=2.0,<2.2
+django>=2.0.6,<2.2
```
Comment (rohithasrk) [inline requirements.txt]: Do we need this change in django version?

File: requirements.txt (hunk 1 above)
Comment (DotSlashTX) [reply requirements.txt]: In a moment!

**Comment by DotSlashTX** [comment]
Please check the PR now :)

**Comment by atb00ker** [comment]
Hi @DotSlashTX , in future, please don't close the pull requests, rather try to fix the same pull request!
 ( It will teach you to use Git properly! )

**Comment by nemesisdesign** [comment]
@DotSlashTX you can update the same pull request by using `git push -f origin <your-dev-branch-here>`

**Pull request closed without merging**